bin/ldapo

```diff
@@ -0,0 +1,284 @@
+#!/usr/bin/env python
+
```

Comment:
@joshmoore for this new script, it works very nicely. But I would appreciate more verbose output.

```
[pwalczysko@ome-training-outreach ldap]$ ./ldapo --apply user public --password public
(105, [])
```
is too little information.
Maybe if I knew how to double check the users already in ldap ...

it could be even behind a ``--verbose`` flag

Reply:
yes, actually the ``./search`` script is giving me the list. But still, it would be much more satisfying to have some informative output from the ``./ldapo`` as well

Reply:
I was surprised at how terse the output from `python-ldap` is. One option would be to run the related search and output that. Happy to add it (either with or without the `--verbose` flag).

Reply:
I think so, without the ``--verbose`` is fine by me.